SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialTraceEventDataBuilder.h

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
```

Comment:
Am I right in thinking the DataBuilder is basically a serialisation routine?

Reply:
I am not familiar with "serialisation routine".

The DataBuilder provides a simple interface for adding data to the event data associated with an event.

Reply:
Basically transforming data so it can be sent / read / stored by some other process.